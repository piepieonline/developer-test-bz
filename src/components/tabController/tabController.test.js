import React from 'react';
import ReactDOM from 'react-dom';
import TabController from './tabController';

const testTabs = [
    {
        "title": "Description",
        "content": "Description."
    },
    {
        "title": "Details",
        "content": "Details."
    }
];

it('renders the expected HTML', () => {
    const div = document.createElement('div');
    ReactDOM.render(<TabController tabs={testTabs} />, div);

    expect(div).toMatchSnapshot();

    ReactDOM.unmountComponentAtNode(div);
});