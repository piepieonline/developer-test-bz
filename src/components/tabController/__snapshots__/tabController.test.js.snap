// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the expected HTML 1`] = `
<div>
  <div>
    <div>
      <button
        class="tab-heading tab-active"
      >
        Description
      </button>
      <button
        class="tab-heading false"
      >
        Details
      </button>
    </div>
    <hr
      class="tab-hr"
    />
    <p>
      Description.
    </p>
  </div>
</div>
`;
